hardhat.config.js

```diff
@@ -27,19 +33,14 @@ module.exports = {
             url: "http://localhost:8545",
         },
         mainnet: {
-            url: "https://eth-mainnet.g.alchemy.com/v2/" + ALCHEMY_API_KEY,
-            chainId: 1,
+            url: "https://eth-mainnet.g.alchemy.com/v2/" + ALCHEMY_API_KEY_MAINNET,
+            accounts: accounts,
```

Comment:
shouldn't this be as per config on the globals.json? this wil now use the wrong key

Reply:
This information is figured out from the default RPC provider when setting the network in the command line. Globals don't use that info in L1 scripts.